corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -62,6 +63,17 @@ def get_active_case_counts_by_owner(domain, datespan, case_types=None, owner_ids
     return _get_case_case_counts_by_owner(domain, datespan, case_types, False, owner_ids)
 
 
+def get_reverse_indexed_cases_es(domain, case_id):
```

Comment:
If this is not used I think it should be possible to remove the changes to the index and associated tests from this pr and move it to another pr if it will be needed later